src/api/tests/feature_tests/features/createCpmProduct.success.feature

```diff
@@ -19,24 +19,25 @@ Feature: Create CPM Product - success scenarios
       | path         | value            |
       | product_name | My Great Product |
     Then I receive a status code "201" with body
-      | path                             | value                                                               |
-      | resourceType                     | OperationOutcome                                                    |
-      | id                               | << ignore >>                                                        |
-      | meta.profile.0                   | https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome |
-      | issue.0.severity                 | information                                                         |
-      | issue.0.code                     | informational                                                       |
-      | issue.0.details.coding.0.system  | https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome |
-      | issue.0.details.coding.0.code    | RESOURCE_CREATED                                                    |
-      | issue.0.details.coding.0.display | Resource created                                                    |
-      | issue.0.diagnostics              | Resource created                                                    |
+      | path            | value                      |
+      | id              | << ignore >>               |
```

Comment:
Given I note the response field "$.id" as "product_id"
${ note(product_id) } can be used instead of << ignore >>